miden/tests/integration/air/stack/mod.rs

```diff
@@ -6,7 +6,7 @@ mod stack_manipualtion_ops;
 /// Test empty starting stack with no overflow outputs.
 #[test]
 fn empty_input() {
-    let asm_op = "push.1 drop";
+    let asm_op = "push.1 drop emit.2";
     let pub_inputs = vec![];
```

Comment:
Should this go into a its own test?

Reply:
Removed in favor of `emit()` in `miden/tests/integrations/operations/sys_ops.rs`